src/main/java/com/gtnewhorizons/angelica/glsm/states/BooleanState.java

```diff
@@ -23,7 +22,7 @@ public void enable() {
     }
 
     public void setEnabled(boolean enabled) {
-        if (GLStateManager.shouldBypassCache() || enabled != this.enabled) {
+        if (GLStateManager.shouldBypassCache() || enabled != this.enabled || (this.glCap == GL11.GL_BLEND && GLStateManager.isPoppingAttributes())) {
```

Comment:
We should be able to also restrict this to AMD as well?  I don't think other drivers have this bug

Reply:
Also does it fix the tests for you locally @Caedis ?

Reply:
That didnt fix it when I tested it last night. Completely disabling GLSM did though

Reply:
I'm only talking about the test that was failing, not the font issue.

Reply:
Yep. All passed
![image](https://github.com/GTNewHorizons/Angelica/assets/18713839/663dd4f9-a661-4350-9f68-18aff96a1c6e)

Reply:
cool, will make this AMD only then

Reply:
What if the font issue is bettermainmenu related?

Reply:
It's some interaction between BLS/BMM and GLSM... I am going to finish fully testing the GLSM (textures) and see if that resolves it, or if not dig into it with api trace